tests/unit/resolvers/classic/basic-test.js

```diff
@@ -265,12 +274,12 @@ test('warns if looking up a camelCase helper that has a dasherized module presen
     return { default: { isHelperInstance: true } };
   });
 
-  Ember.warn = function(message, test, options) {
+  this.sandbox.stub(Ember, 'warn').callsFake((message, test, options) => {
```

Comment:
I think we can also replace this with `assert.expectWarning`

Reply:
Refactored this, it is much cleaner with those helpers. One thing I found with the helper is we are not able to reach in and assert against the warning's id.